### PR TITLE
Bug wave2 718 field standarization special restricted bricks

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_spl/ucgy_spl_impact_stories/ucgy_spl_impact_stories.module [hunk 1]
```diff
@@ -48,3 +50,37 @@ function ucgy_spl_impact_stories_theme($existing, $type, $theme, $path)
...
+  }
+
+  // print_r($form);
+}
+
+function field_ucimp_link_after_build($form, &$form_state) {
+  foreach ($form as $k => $v) {
+    if (isset($v['#entity_type']) && $v['#entity_type'] == 'fieldable_panels_pane') {
+
+      $form[0]['title']['#title'] = 'Link label';
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_spl/ucgy_spl_impact_stories/ucgy_spl_impact_stories.module]: Should be wrapped in t()

File: sites/all/modules/contrib/manualcrop/includes/views/manualcrop.views.inc [hunk 2]
```diff
@@ -8,7 +8,7 @@
  * Implements hook_views_data().
 */
 function manualcrop_views_data() {
-  $data['manualcrop']['table']['group'] = t('Manual Crop');
+  $data['manualcrop']['table']['group'] = t('Manual crop');
```
Comment (mhughes2012) [inline sites/all/modules/contrib/manualcrop/includes/views/manualcrop.views.inc]: fixed this. reverted all the manualcrop module updates and made this copy change via string overrides

File: sites/all/modules/ucalgary/ucgy_spl/ucgy_spl_impact_stories/ucgy_spl_impact_stories.module (hunk 1 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_spl/ucgy_spl_impact_stories/ucgy_spl_impact_stories.module]: wrapped now in t(). Sorry about that.

File: sites/all/modules/contrib/manualcrop/manualcrop.module [hunk 3]
```diff
@@ -254,6 +254,9 @@ function manualcrop_field_widget_info_alter(&$info) {
  * Implements hook_field_widget_form_alter().
  */
 function manualcrop_field_widget_form_alter(&$element, &$form_state, $context) {
+
```
Comment (dpfitzsi) [inline sites/all/modules/contrib/manualcrop/manualcrop.module]: Looks like three extra spaces got added here. If they can be removed this should be good to go!

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/test [hunk 4]
```diff
@@ -0,0 +1,21 @@
+Array (
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/test]: This should also be removed.

**Comment by mhughes2012** [comment]
did the cleanup

File: sites/all/modules/contrib/manualcrop/manualcrop.module (hunk 3 above)
Comment (mhughes2012) [inline sites/all/modules/contrib/manualcrop/manualcrop.module]: cleaned up.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/test (hunk 4 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/test]: removed now.

**Pull request merged**